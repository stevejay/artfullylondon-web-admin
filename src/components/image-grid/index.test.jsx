import React from 'react'
import { shallow } from 'enzyme'

import ImageGrid from '_src/components/image-grid'

it('should render correctly when has children', () => {
  const wrapper = shallow(<ImageGrid><div id='child' /></ImageGrid>)
  expect(wrapper).toMatchSnapshot()
})

it('should render correctly when has no children', () => {
  const wrapper = shallow(<ImageGrid />)
  expect(wrapper).toMatchSnapshot()
})
