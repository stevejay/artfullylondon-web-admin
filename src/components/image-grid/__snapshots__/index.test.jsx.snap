// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly when has children 1`] = `
<div
  className="src-components-image-grid-index__container"
>
  <div
    id="child"
  />
</div>
`;

exports[`should render correctly when has no children 1`] = `
<NoEntries
  label="No Entries"
/>
`;
